### PR TITLE
Define and activate release 16.2.1 for kvm

### PR DESCRIPTION
**Pull request by pipo02mix**

### CODE REVIEW

File: kvm/v16.2.1/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# :zap: Giant Swarm Release v16.2.1 for KVM :zap:
+
+This release updates several preinstalled apps running in the workload cluster to reduce Kubernetes API server load and etcd memory usage in certain installations.
```
Comment (kopiczko) [inline kvm/v16.2.1/README.md:3]: I think this needs update

Comment (pipo02mix) [inline kvm/v16.2.1/README.md:3]:
```suggestion
This release enables the cluster to define multiple registry mirrors.
```

File: kvm/v16.2.1/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# :zap: Giant Swarm Release v16.2.1 for KVM :zap:
+
+This release enables the cluster to define multiple registry mirrors.
+
+## Change details
+
+
+### kvm-operator [3.18.6](https://github.com/giantswarm/app-operator/releases/tag/v3.18.6)
```
Comment (pipo02mix) [inline kvm/v16.2.1/README.md:8]:
```suggestion
### kvm-operator [3.18.6](https://github.com/giantswarm/kvm-operator/releases/tag/v3.18.6)
```

File: kvm/v16.2.1/README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# :zap: Giant Swarm Release v16.2.1 for KVM :zap:
+
+This release enables the cluster to define multiple registry mirrors.
```
Comment (pipo02mix) [inline kvm/v16.2.1/README.md:3]:
```suggestion
This release enables the workload clusters to define multiple registry mirrors.
```

**Review by kopiczko — APPROVED** [review]

**Pull request merged**